pyquil/paulis.py

```diff
@@ -318,7 +318,7 @@ def compact_str(self):
         return f'{self.coefficient}*{self.id(sort_ops=False)}'
 
     @classmethod
-    def from_list(cls, terms_list, coefficient=1.0):
+    def from_list(cls, terms_list, coefficient: float = 1.0) -> 'PauliTerm':
```

Comment:
```suggestion
    def from_list(cls, terms_list: List[Tuple(str, int)], coefficient: float = 1.0) -> 'PauliTerm':
```

Actually, `Iterable` would be preferable to `List`, but since the method is called `from_list`, go for consistency I guess.